README.rst

```diff
@@ -82,20 +99,43 @@ Once you have an environment set up, install our Python package requirements:
     pip install -r requirements/local.txt
 
 
+Fedora Linux environment
+------------------------
+
+The following installation method has been tested on Fedora 26. Some packages
+must be installed on the system level and are available via ``dnf``, the python
+specific dependencies can be installed into a virtual environment.
+
+.. code-block:: bash
+
+    sudo dnf install gettext postgresql postgresql-devel python3 libffi-devel libffi
+    python3 -m venv ~/envs/cloudigrade
+    source ~/envs/cloudigrade/activate
+    pip install -U pip awscli wheel tox 
```

Comment:
Is there a good reason why we can't just include awscli wheel and tox in the requirements or in some setup.py?